File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex16.c
#include <stdio.h>

main()
{
    printf("N Primeiros Primos\n\n");

    //vars
    int n, num=2, divisor, contPrimo, contQtd=0;

    //Entrada
    printf("Digite a quantidade de numeros primos: ");
    scanf("%d",&n);

    while(contQtd != n)
    {
        contPrimo = 0;
        for(divisor=1; divisor<=num; divisor++)
        {
            if(num%divisor == 0)
                contPrimo++;
        }

        if(contPrimo == 2)
        {
            printf("%d\n", num);
            contQtd++;
        }

        num++;

    }

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex16.java
//Importando Scanner
import java.util.Scanner;

public class Ex16 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("Números Primos\n\n");
		
		//vars
		int n=2, num, divisor, contPrimo, contTermos=1;
		
		//Entrada
		System.out.print("Digite a quantidade de números primos: ");
		num = input.nextInt();
		
		//Processamento
		while(  contTermos<=num )
		{
			contPrimo = 0;
			for( divisor=1; divisor<=n; divisor++ )
			{
				if( n%divisor == 0 )
					contPrimo++;
			}
			
			if( contPrimo == 2 )
			{
				contTermos++;
				System.out.println(n);
			}
			
			n++;
			
		}
		
	}
}
